### PR TITLE
Add docker-compose

### PR DESCRIPTION
**Pull request by prtngn**
Removed:
- old Dockerfile.arm64v8

Added:
- docker-compose.yml to compose 0net and tor containers separately.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,21 +1,17 @@
...
 
-ENV ENABLE_TOR false
-
-CMD (! ${ENABLE_TOR} || tor&) \
- && source venv/bin/activate \
- && python3 zeronet.py --ui_ip "*" --fileserver_port 26552
+CMD source venv/bin/activate \
+ && python3 zeronet.py --ui_ip "*" --fileserver_port 26552 \
+    --tor $TOR_ENABLED --tor_controller tor:$TOR_CONTROL_PORT \
+    --tor_proxy tor:$TOR_SOCKS_PORT --tor_password $TOR_CONTROL_PASSWD
```
Comment (caryoscelus) [inline Dockerfile:15]: as per #92 and #96 , it would make sense to append `main` to make sure the password doesn't collide with another command

**Comment by caryoscelus** [comment]
@FraYoshi would you like to test this before i merge?

**Review by caryoscelus — APPROVED** [review]

**Comment by FraYoshi** [comment]
`docker-compose` is okay and works fine instead 👍🏻

**Review by FraYoshi — APPROVED** [review]
apart from the readme, it can be merged, thanks prtngn!

**Pull request merged**